Naggum.Assembler/Assembler.fs

```diff
@@ -0,0 +1,65 @@
+﻿module Naggum.Assembler.Assembler
+
+open System
+open System.Reflection
+open System.Reflection.Emit
+
+open Naggum.Assembler.Representation
+
+let private getMethodAttributes (m : MethodDefinition) =
+    let empty = enum 0
+    let conditions =
+        [ (m.Visibility = Public, MethodAttributes.Public)
+          (true, MethodAttributes.Static) ] // TODO: Proper static method detection
+
+    conditions
+    |> List.map (fun (c, r) -> if c then r else empty)
+    |> List.fold (|||) empty
+
+let private findMethod (signature : MethodSignature) =
+    // TODO: Add method overload resolution etc. (see ClrGenerator module)
```

Comment:
Should we really add method overload resolution here? I doubt: here the user has written the method exactly as he wants it to be resolved, without any compiler-involved resolution.